CONTRIBUTING.md

```diff
@@ -114,40 +114,12 @@ of each `.h` and `.cpp` file:
 
 ```cpp
 /*
- * Software License Agreement (BSD License)
+ * SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Why did you use BSD-3-Clause and not one of the other BSD variants?

Reply:
Because our [License](https://github.com/PointCloudLibrary/pcl/blob/master/LICENSE.txt) is BSD with 3 clauses.

[Link to Wikipedia for comparison](https://en.wikipedia.org/wiki/BSD_licenses#3-clause_license_(%22BSD_License_2.0%22,_%22Revised_BSD_License%22,_%22New_BSD_License%22,_or_%22Modified_BSD_License%22))

Reply:
SPDX requires being exact about the license version since the entire text is not available